src/realm-settings/ExecutorForm.tsx

```diff
@@ -178,26 +260,67 @@ export default function ExecutorForm() {
                 );
               }
             })}
+            {editMode &&
+              editedProfileExecutors?.map((option) => {
+                const componentType = option.type!;
+                if (isValidComponentType(componentType)) {
+                  const Component = COMPONENTS[componentType];
+                  return (
+                    <Component
+                      key={option.name}
+                      {...option}
+                      name={option.name}
+                      label={option.label}
+                    />
+                  );
+                } else {
+                  console.warn(
+                    `There is no editor registered for ${componentType}`
+                  );
+                }
+              })}
           </FormProvider>
-          <ActionGroup>
-            <Button
-              variant="primary"
-              onClick={save}
-              data-testid="addExecutor-saveBtn"
-            >
-              {t("common:add")}
-            </Button>
+          {!globalProfile ? (
+            <ActionGroup>
+              <Button
+                variant="primary"
+                onClick={() => handleSubmit(save)()}
+                data-testid="addExecutor-saveBtn"
+              >
+                {editMode ? t("common:save") : t("common:add")}
+              </Button>
+              <Button
+                variant="link"
+                component={(props) => (
+                  <Link
+                    {...props}
+                    to={toClientProfile({ realm, profileName })}
+                  />
+                )}
+                data-testid="addExecutor-cancelBtn"
+              >
+                {t("common:cancel")}
+              </Button>
+            </ActionGroup>
+          ) : undefined}
```

Comment:
There is no need to use a ternary operation here.
```suggestion
          {!globalProfile && (
            <ActionGroup>
              <Button
                variant="primary"
                onClick={() => handleSubmit(save)()}
                data-testid="addExecutor-saveBtn"
              >
                {editMode ? t("common:save") : t("common:add")}
              </Button>
              <Button
                variant="link"
                component={(props) => (
                  <Link
                    {...props}
                    to={toClientProfile({ realm, profileName })}
                  />
                )}
                data-testid="addExecutor-cancelBtn"
              >
                {t("common:cancel")}
              </Button>
            </ActionGroup>
          )}
```